packages/trader/src/Modules/Trading/Components/Form/TradeParams/Duration/duration-wrapper.jsx

```diff
@@ -5,63 +5,65 @@ import { connect } from 'Stores/connect';
 import { getDurationMinMaxValues } from 'Stores/Modules/Trading/Helpers/duration';
 import Duration from './duration.jsx';
 
-class DurationWrapper extends React.Component {
-    hasDurationUnit = (duration_unit, is_advanced_duration) => {
-        let duration_list = [...this.props.duration_units_list];
-
-        if (duration_list.length > 1 && !is_advanced_duration) {
+const DurationWrapper = props => {
+    const {
+        advanced_duration_unit,
+        advanced_expiry_type,
+        duration,
+        duration_unit,
+        expiry_type,
+        getDurationFromUnit,
+        is_advanced_duration,
+        onChange,
+        onChangeUiStore,
+        simple_duration_unit,
+        contract_expiry_type,
+        duration_min_max,
+        duration_units_list,
+        onChangeMultiple,
+    } = props;
```

Comment:
Let's just use this extraction on line 8 itself.

Reply:
I need to pass `props` to `Duration` component on line `134`